packages/invoices/src/OutboundShipment/DetailView/modals/ItemDetailsForm.tsx

```diff
@@ -77,7 +78,7 @@ export const ItemDetailsForm: React.FC<ItemDetailsFormProps> = ({
       <ModalRow>
         <ModalLabel label={t('label.available')} />
         <NumericTextInput
-          value={availableQuantity}
+          value={Math.max(availableQuantity - allocatedQuantity, 0)}
```

Comment:
it does mean that this can happen:
- available quantity = 100
- assign 500 to a placeholder line
- available : 0 allocated : 500

that makes sense to me, there's no more available. seems better than showing -400 available 🤷